habitat-lab/habitat/sims/habitat_simulator/sim_utilities.py

```diff
@@ -754,6 +754,74 @@ def within(
     return containment_ids
 
 
+def ontop(
+    sim: habitat_sim.Simulator,
+    objectA: Union[
+        habitat_sim.physics.ManagedRigidObject,
+        habitat_sim.physics.ManagedArticulatedObject,
+        int,
+    ],
+    do_collision_detection: bool = True,
```

Comment:
Beware that this default value may cause performance issues.

Folks may start using this function liberally to query the scene.

Reply:
Good point, we can remove the default here.